package.json

```diff
@@ -20,13 +20,13 @@
   "scripts": {
     "lint": "yarn eslint ./src --ext .ts,.tsx",
     "postinstall": "node ./scripts/postinstall.js",
-    "build:prod": "rimraf dist && webpack && yarn pack:prod",
+    "build:prod": "rimraf dist && webpack",
     "build:dev": "rimraf dist && webpack --env dev",
-    "pack:prod": "vsce package --githubBranch main --no-dependencies --yarn -o ./dist/vscode_kaoto_$npm_package_version.vsix",
     "compile": "webpack",
     "watch": "webpack --env dev",
     "run:webmode": "yarn vscode-test-web --browserType=chromium --extensionDevelopmentPath=. --open-devtools ./resources",
     "test:it": "tsc --project tsconfig.it-tests.json --skipLibCheck --sourceMap true && extest setup-and-run --yarn --uninstall_extension --extensions_dir ./test-resources 'out/*.test.js'",
+    "test:it:with-prebuilt-vsix": "extest get-vscode && extest get-chromedriver && extest install-vsix --vsix_file vscode-kaoto-$npm_package_version.vsix --extensions_dir ./test-resources && extest run-tests --extensions_dir ./test-resources 'out/*.test.js'",
```

Comment:
@mlorinc I created a separated script to be able to launch test with packed with production on CI

Reply:
The existing script was not deleting the --extension-dir so sounds fine to keep the same behavior.

Reply:
test:it has --uninstall_extension, so it is removed in background. Directory itself is not removed, however the extension is.

Reply:
I think we cannot use --uninstall_extension otherwise, it will remove the one that we have just installed and want to test

Reply:
It will be uninstalled in after all Mocha Hook (handled by extester).

https://github.com/redhat-developer/vscode-extension-tester/blob/672c58cf995924df748911414e12bc2a69c331e9/src/suite/runner.ts#L91

https://github.com/redhat-developer/vscode-extension-tester/blob/672c58cf995924df748911414e12bc2a69c331e9/src/util/codeUtil.ts#L211